vesc_hw_interface/src/vesc_wheel_controller.cpp

```diff
@@ -232,11 +234,18 @@ void VescWheelController::updateSensor(const std::shared_ptr<const VescPacket>&
     std::shared_ptr<VescPacketValues const> values = std::dynamic_pointer_cast<VescPacketValues const>(packet);
     const double current = values->getMotorCurrent();
     const double velocity_rpm = values->getVelocityERPM() / static_cast<double>(num_rotor_pole_pairs_) * gear_ratio_;
+    prev_steps_ = steps_;
     steps_ = values->getPosition();
```

Comment:
`steps_` is used as relative position in `control()`.
https://github.com/sbgisen/vesc/blob/e5817aa7eb2b986d358cd3bdf2a59e2e4c1ab08f/vesc_hw_interface/src/vesc_wheel_controller.cpp#L225

Reply:
@nyxrobotics Any words on this? I believe the original source code used "absolute" position hence I left it as it is. If this should be a relative position, then I will change it.

Reply:
Since it is initialized with the reset flag, the behavior does not change either way, but I think it is easier to see if it is unified with relevant.

Reply:
> `steps_` is used as relative position in `control()`.
> 
> https://github.com/sbgisen/vesc/blob/e5817aa7eb2b986d358cd3bdf2a59e2e4c1ab08f/vesc_hw_interface/src/vesc_wheel_controller.cpp#L225

Forget to mention this; the reset process is called first before control starts to make sure the target_steps are relative to current_steps, so the above code should be ok. 
https://github.com/sbgisen/vesc/blob/f8d522070c3a44035a2e4a76c029da9296adf1de/vesc_hw_interface/src/vesc_wheel_controller.cpp#L57-L59